### PR TITLE
Fix bug where the decompressor was not appending rc1 to filename

### PR DESCRIPTION
**Pull request by dakersnar**

### CODE REVIEW

File: src/tools/ResultsComparer/Data.cs [hunk 1]
```diff
@@ -167,4 +169,4 @@ private static string GetSimplifiedProcessorName(string processorName)
             return processorName.Replace(" ", "");
         }
     }
-}
\ No newline at end of file
+}
```
Comment (mrsharm) [inline src/tools/ResultsComparer/Data.cs:172]: Weird that this closing paren shows up. Can we check if the scripts still work to prove that this was unintentional?

**Review by EgorBo — APPROVED** [review]

File: src/tools/ResultsComparer/Data.cs (hunk 1 above)
Comment (EgorBo) [reply src/tools/ResultsComparer/Data.cs:172]: it's a newline at the end of the file (added)

**Comment by EgorBo** [comment]
I guess it could be just a simple Regex (to handle potentially new suffixes)

**Review by mrsharm — APPROVED** [review]
LGTM!

File: src/tools/ResultsComparer/Data.cs (hunk 1 above)
Comment (dakersnar) [reply src/tools/ResultsComparer/Data.cs:172]: Yeah, I'm not sure why that newline was added. Every time I tried to delete it it comes back. Is it a problem?

Comment (mrsharm) [reply src/tools/ResultsComparer/Data.cs:172]: I don't think it should be a problem. One quick check you could do is rerun the script, however, as per @EgorBo's comment, I think we should be good.

**Pull request merged**